test/parallel/test-repl-envvars.js

```diff
@@ -72,4 +81,4 @@ function run(test) {
   });
 }
 
-tests.forEach(run);
+tests.forEach((testCase) => test(inspect(testCase.env), () => run(testCase)));
```

Comment:
tests did not play well when running in parallel. used `node:test` to run them sequentially